me.ahola.aphototoollibre.yml

```diff
@@ -0,0 +1,27 @@
+app-id: me.ahola.aphototoollibre
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: aphototoollibre
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --env=XDG_SESSION_TYPE=x11
+  - --filesystem=host
+  - --device=dri
+modules:
+  - name: aphototoollibre
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/aphototool/A-Photo-Tool-Libre/archive/refs/tags/v1.0.5-3.tar.gz
+        sha256: f28201302d36c510e542e7d11653afddba0a2df4a829bc5e810f7ed5c365b5d3
+    build-commands:
+      - qmake
```

Comment:
it should be `buildsystem: qmake`

Reply:
Yes, but that will require work on my .pro file that I like to make later.

Reply:
why so?